docs/docs/hosting/hub-setup.mdx

```diff
@@ -0,0 +1,33 @@
+---
+sidebar_position: 2
+---
+
+# Hub setup
+
+## Setting an app key
+
+`APP_KEY` must be set to a secret, unguessable value. The value must be 32
```

Comment:
or set it with `php artisan key:generate`

Reply:
If used with the option `--show` it will only display the key without modifying files
`php artisan key:generate --show`

Reply:
This assumes the setup is done in such a way that they can run artisan commands while they're setting up Edlib. If ssh'ing into a server, that's likely the case, but in automated deployments that assumption falls apart. For hosting purposes, it's better to describe how this app key is constructed without invoking artisan, so whoever's reading the documentation can figure out a way of generating these keys that works for them.